pkg/template/helper.go

```diff
@@ -28,7 +28,9 @@ func updateExpectedValue(template TestTemplate) {
 	}
 }
 
-// executeTestCombination get a template and pass it to `processTestCombination` to execute test combination on group of IPs
+// executeTestCombination get a template and pass it to `processTestCombination`
+//
```

Comment:
Is it best practice to have an "empty new line" like this?

Reply:
for bigger doc comments yes, so they get formatted on documentation